uaclient/entitlements/fips.py

```diff
@@ -301,49 +301,53 @@ def packages(self) -> List[str]:
         packages = super().packages
         return self._replace_metapackage_on_cloud_instance(packages)
 
+    def _notice_remove(self, label: str, description: str):
+        try:
+            self.cfg.notice_file.remove(label, description)
+        except exceptions.NonRootUserError:
+            pass
```

Comment:
I'd rather see this function on the notice_file itself, so it can be used in other places if needed.

It could look like this:
```python
notice_file.remove() # raises NonRootUserError when not root
notice_file.try_remove() # logs warning but doesn't raise error when not root
```

Reply:
for example, I think we need to catch the NonRootUserError in cli.py line 1557